config/settings/base.py

```diff
@@ -234,8 +234,15 @@
 
 SESSION_COOKIE_SECURE = True
 
+# Force browser to end session when closing.
 SESSION_EXPIRE_AT_BROWSER_CLOSE = True
 
+# Since some browser restore session when restarting, the previous setting may not
+# work as we want. This is why we ask Django to expire sessions in DB after 1 week
+# of inactivity.
+# -> https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_the_lifetime_of_a_cookie
```

Comment:
Je rajouterais bien un commentaire du genre:
`# In addition, the command shorten_active_sessions is run every week to force user to connect at least once per week`

Reply:
excellente idée, j'ai ajouté